microsoft-365/compliance/information-risk-management-reference-architecture.md

```diff
@@ -0,0 +1,158 @@
+
+## Audits and Legal Litigation Management
+
+The [unified auditing](advanced-audit.md) functionality in Microsoft 365 provides organizations with visibility into many types of audited activities across many different services in Microsoft 365. Advanced Audit helps organizations to conduct forensic and compliance investigations by increasing audit log retention required to conduct an investigation, providing access to crucial events that help determine scope of compromise, and faster access to Office 365 Management Activity API.
+[Advanced eDiscovery](get-started-with-advanced-ediscovery.md) solution in Microsoft 365 builds on the existing Microsoft eDiscovery and analytics capabilities. Advanced eDiscovery provides an end-to-end workflow to preserve, collect, analyze, review, analyze, and export content that's responsive to your organization's internal and external investigations. It also lets legal teams manage the entire legal hold notification workflow to communicate with custodians involved in a case.
+
+## Crypto Key Management Management
+
+For extremely regulated environment and subset of data that is highly sensitive, [Double Key Encryption (DKE)](double-key-encryption.md) offers the use of two keys together to access protected content. Microsoft stores one key in Microsoft Azure, and you hold the other key. You maintain full control of one of your keys using the Double Key Encryption service. You apply protection using The Azure Information Protection unified labeling client to your highly sensitive content.
+
+Organizations with an MIP subscription can choose to configure their tenant with their own key, instead of a default key generated by Microsoft. This configuration is often referred to as [Bring Your Own Key (BYOK)](customer-key-overview.md). BYOK and usage logging work seamlessly with applications that integrate with the Azure Rights Management service used by MIP, protecting the top few percent of your most sensitive information in a manner that is opaque to anyone but you. It is linked to your organization’s requirement of maintaining full control over the encryption keys and over the authorization process for the ‘top-secret’ data. However, it does come at the price of limiting any service that can reason over data or protect your data flow outside of the organization perimeter.
```

Comment:
```suggestion
Organizations with an MIP subscription can choose to configure their tenant with their own key, instead of a default key generated by Microsoft. This configuration is often referred to as [Bring Your Own Key (BYOK)](customer-key-overview.md). BYOK and usage logging work seamlessly with applications that integrate with the Azure Rights Management service used by MIP, protecting the top few percent of your most sensitive information in a manner that is opaque to anyone but you. It is linked to your organization’s requirement of maintaining full control over the encryption keys and over the authorization process for the ‘top-secret’ data. However, it does come at the price of limiting any service that can reason over data or protect your data flow outside of the organization's perimeter.
```